iet.so/src/iet_worker.cpp

```diff
@@ -258,7 +258,7 @@ bool IETWorker::do_iet_power_stress(void) {
         }
 
         //It doesnt make sense to read power continously so slowing down
-        sleep(1000);
+        sleep(sample_interval);
```

Comment:
Would we need to use this_thread::sleep() instead now that we are making a fine grained change? sleep() is not thread aware.

Reply:
sleep_for() used instead of usleep